main/src/ca/uwaterloo/flix/language/errors/TypeError.scala

```diff
@@ -63,26 +63,36 @@ object TypeError {
   /**
    * Java method not found type error.
    *
-   * @param oTpe    the type of the receiver object.
-   * @param mTpe    the type of the method, i.e., of its arguments.
+   * @param tpe0    the type of the receiver object.
+   * @param tpes    the types of the arguments.
    * @param methods a list of possible candidate methods on the type of the receiver object.
    * @param renv    the rigidity environment.
    * @param loc     the location where the error occured.
    */
-  case class MethodNotFound(methodName: String, oTpe: Type, mTpe: Type, methods: List[JvmMethod], renv: RigidityEnv, loc: SourceLocation)(implicit flix: Flix) extends TypeError {
+  case class MethodNotFound(methodName: String, tpe0: Type, tpes: List[Type], methods: List[JvmMethod], renv: RigidityEnv, loc: SourceLocation)(implicit flix: Flix) extends TypeError {
     // TODO INTEROP better comment, e.g., list possible methods
-    def summary: String = s"Java method with type '$mTpe' in type '$oTpe' not found."
+    def summary: String = s"Java method '$methodName' in type '$tpe0' with arguments types '${tpes.toString}' not found."
 
     def message(formatter: Formatter): String = {
       import formatter._
       s"""${line(kind, source.name)}
-         |>> Java method '$methodName' with signature '${red(formatType(mTpe, Some(renv)))}' from '${red(formatType(oTpe, Some(renv)))}' not found.
+         |>> Java method '$methodName' from type '${red(formatType(tpe0, Some(renv)))}' with arguments types '${tpes.foldLeft("") { (acc, t) => acc + red(formatType(t, Some(renv))) }}' not found.
```

Comment:
I think this won't look right. We need some commas and spaces between the types.

I suggest doing `tpes.map(...).mkString(", ")` to simplifty formatting